extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventActionExecutionFieldsMixin.schema.json

```diff
@@ -123,6 +123,105 @@
               "type": "integer",
               "meta:titleId": "journeyStepEventActionExecutionFields##https://ns.adobe.com/experience/journeyOrchestration/actionSchedulerCount##title##88741",
               "meta:descriptionId": "journeyStepEventActionExecutionFields##https://ns.adobe.com/experience/journeyOrchestration/actionSchedulerCount##description##46891"
+            },
+
+            "https://ns.adobe.com/experience/journeyOrchestration/messageID": {
+                "title": "MessageID",
+                "description": "Message ID referenced in inline message action of a journey",
+                "type": "string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/publishedMessageID": {
+                "title": "PublishedMessageID",
+                "description": "Published message ID in inline message action of a journey",
+                "type": "string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/interactionType": {
+                "title": "InteractionType",
+                "description": "Indicate the interaction type, either 'marketing' or 'transactional'. Remark: this will be relevant for message and custom actions",
+                "type": "string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/surfaceID": {
+                "title": "SurfaceID",
+                "description": "Surface ID",
+                "type": "string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/channel": {
```

Comment:
Same as above...enum ?

Reply:
Hey @mjasrotia , thanks for reviewing. We discussed it in our team and it does make sense to eventually make it enum, but for now it is too early and we would want to keep it like this for sometime as eventually the naming might be a bit different when hardened and more values could be added.